src/libraries/System.IO.FileSystem/tests/File/GetSetAttributes_SafeFileHandle.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace System.IO.Tests
+{
+    public sealed class GetSetAttributes_SafeFileHandle : File_GetSetAttributes
```

Comment:
`GetSetAttributes_SafeFileHandle` extends `File_GetSetAttributes` 
but 
`File_GetSetAttributes_String` extends `FileGetSetAttributes`?

The test folders/files that we have in IO are such a mess.